pages/page.py

```diff
@@ -35,9 +34,7 @@ def __init__(self, testsetup):
     def is_the_current_page(self):
         if self._page_title:
             page_title = self.page_title
-            Assert.equal(page_title, self._page_title,
-                         "Expected page title: %s. Actual page title: %s" %
-                         (self._page_title, page_title))
+            assert page_title == self._page_title
```

Comment:
I think this isn't the forum for this discussion, that said I agree with the move to the new `asserts`. From a BDD approach I am not in love with the new error messages.